components/Publish.vue

```diff
@@ -160,6 +177,42 @@ export default {
       });
       return false;
     },
+    async wrap() {
+      const signer = await this.provider.getSigner();
+      const signerAddr = await signer.getAddress();
+      if (signerAddr.toLowerCase() != this.ad.owner) {
+        this.error = 'Incorrect active wallet. Must publish with: ' + this.ad.owner;
+        return;
+      }
+      try {
+        const { predictedAddress } = (await this.ketherNFT.precompute(this.ad.idx, signerAddr));
```

Comment:
let's add an assert here that it's not a 0-address.

Reply:
If we wanna be very silly and double-check the code client-side, we can do https://github.com/thousandetherhomepage/ketherhomepage/blob/7774eb9d6fb58e28086bfbbcc095d9878ffd65e3/test/kethernft.js#L303-L315 (imo not worth it)